pkg/model/requests_processing_test.go

```diff
@@ -68,24 +69,71 @@ func testRequestsProcessing(t *testing.T, describe spec.G, it spec.S) {
 			assert.Equal(t, simulator.StockName("RequestsProcessing [99]"), subject.Name())
 		})
 	})
-
 	describe("Add()", func() {
-		var request simulator.Entity
+		var request RequestEntity
 
-		it.Before(func() {
-			bufferStock := NewRequestsRoutingStock(envFake, NewReplicasActiveStock(), nil)
-			request = NewRequestEntity(envFake, bufferStock, RequestConfig{CPUTimeMillis: 200, IOTimeMillis: 200, Timeout: 3 * time.Second})
-			subject.Add(request)
+		describe("there is no free cpu resource", func() {
+			it.Before(func() {
+				*rawSubject.occupiedCPUCapacityMillisPerSecond = *rawSubject.totalCPUCapacityMillisPerSecond
+				routingStock := NewRequestsRoutingStock(envFake, NewReplicasActiveStock(), nil)
+				request = NewRequestEntity(envFake, routingStock, RequestConfig{CPUTimeMillis: 200, IOTimeMillis: 200, Timeout: 3 * time.Second})
+				subject.Add(request)
+			})
+			describe("scheduling processing", func() {
+				it("schedules a movement from RequestsProcessing to RequestsFailed", func() {
+					assert.Equal(t, simulator.StockName("RequestsFailed"), envFake.Movements[0].To().Name())
+				})
+			})
+
+			it("there is 1 entity in processingStock", func() {
+				assert.Equal(t, subject.Count(), uint64(1))
```

Comment:
Since ["expected" goes first](https://godoc.org/github.com/stretchr/testify/assert#Equal), perhaps better to stick to it: assert.Equal(t, uint64(1), subject.Count())

Nit, but there might be a difference in messaging, e.g. if subject.Count() is 2, it would be "2 was expected to be 1" (more clear what's going on) vs. "1 was expected to be 2" (confusing)

Reply:
Fixed it. (see commit 2d6552efb24f1c1eed205483f07717a4798bf296)